src/components/Footer/StyledFooter.js

```diff
@@ -12,8 +12,10 @@ const ContactPanelRow = styled(Row)``
 
 // Individual social media contact column
 const ContactPanelColumn = styled(Col)`
-  justify-content: center;
-  padding: 30px;
+${genericStyles}
```

Comment:
I don't think we need to import `genericStyles` here since its already inherited from the `Col` component